src/navigate-pop/tiling-stacking-windows.md

```diff
@@ -0,0 +1,107 @@
+
+![Enable Autotiling](/images/tiling-stacking-windows/enable-autotiling.png)
+
+| Option                 | Function |
+|----------------------|--------|
+| Tile Windows| Toggles the Auto Tiling feature on or off.  |
+| Floating Window Exceptions  | Allows adding exceptions to the Auto Tiling feature allowing free positioning of specified windows.|
+| Toggle Tiling          | Enables or disables the Auto Tiling feature with a keyboard shortcut. |
+| Show Active Hint       | Highlights the selected window with a colored border. |
+| Gaps                | Sets gap spacing between tiled windows. |
+
+All keyboard shortcuts for Auto Tiling mode can be found in Settings ➞ Keyboard ➞ View and Customize Shortcuts ➞ Tiling.
+
+### Add a Floating Window Exception
+
+Floating Window Exceptions exclude individual windows from Auto Tiling. Pop!\_OS applies default exceptions to some applications based on user feedback.
+
+1. Run the application that needs to be exempted from Auto Tiling. Click the tile menu, then click `Floating Window Exceptions`.
+
+    ![Floating Window Exceptions](/images/tiling-stacking-windows/floating-window-exceptions.png)
+
+2. Click `Select` to add an exception.
+
+    ![Click Select](/images/tiling-stacking-windows/click-select.png)
+
+3. Select the running application that should be exempted from Auto Tiling.
+
+    ![Select Exempted App](/images/tiling-stacking-windows/select-exempted-app.png)
+
+4. Choose to create an exception for all windows belonging to an application, or only the currently selected window.
+
+    - **Current Window Only**: This option uses the window's title to apply the exception in addition to its application, so any windows with the same title (now or in the future) will all have the exception applied.
+    - **This App's Window**: This option will apply the exception to all windows launched by a specified application.
+
+    In this example, only the current Terminal window will be selected for exemption.
+
+    ![Current Window Only](/images/tiling-stacking-windows/current-window-only.png)
+
+5. The Terminal window can now be dragged around freely.
+
+    ![Drag Shell Freely](/images/tiling-stacking-windows/drag-shell-freely.png)
+
+## Stacking Windows
+
+Stacking is an additional Auto Tiling feature enabling multiple windows to occupy the same screen space.
+
+1. While Auto Tiling is enabled, select a window and press `SUPER` + `S` to create a stack. Stacks have a yellow list of windows at the top.
+
+    <video autoplay loop>
+        <source src="/images/tiling-stacking-windows/enable-stacking.webm" />
+    </video>
+
+2. Select a separate window and press `SUPER` + `Enter` to enter adjustment mode. The window to be merged is designated by a yellow highlight across the entire window.
+
+    <video autoplay loop>
```

Comment:
Most of these look great motion-wise, but this one's looping a little awkwardly. I would give it an extra second after the highlight appears before the end where it loops, and also get rid of the unnecessary motion with the clock and Nautilus tab bar changing at the beginning of the video. (If it's more convenient, this stuff should be easy to doctor in Kdenlive without re-recording.)